### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by mikeller**
Ignore this branch - this is where the Crowdin platform that is used by the project pushes all translation updates automatically - in this project translations are added to the repository manually before releases, so this branch is not needed and will never be merged.

### CODE REVIEW

File: locales/nl/messages.json [hunk 1]
```diff
@@ -4140,6 +4152,10 @@
         "message": "Metriek stelsel",
         "description": "Option for the units system used in the OSD"
     },
+    "osdSetupUnitsOptionBritish": {
+        "message": "Brits",
```
Comment (haslinghuis) [inline locales/nl/messages.json:4156]:
```suggestion
        "message": "Brits stelsel",
```

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed (4/06/2021 13:58:38) -> PASS
 - commit count less or equal to three (492 commits) -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - 'Needs coordination with betaflight' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver (None) -> FAIL
 - approver count at least three (0 approvers) -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed (10/06/2021 01:11:34) -> PASS
 - commit count less or equal to three (492 commits) -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - 'Needs coordination with betaflight' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver (None) -> FAIL
 - approver count at least three (0 approvers) -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed (10/06/2021 01:18:53) -> PASS
 - commit count less or equal to three (492 commits) -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - 'Needs coordination with betaflight' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver (None) -> FAIL
 - approver count at least three (0 approvers) -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed (10/06/2021 01:24:01) -> PASS
 - commit count less or equal to three (492 commits) -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver (None) -> FAIL
 - approver count at least three (0 approvers) -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed (10/06/2021 06:27:45) -> FAIL
 - commit count less or equal to three (492 commits) -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver (None) -> FAIL
 - approver count at least three (0 approvers) -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three (492 commits) -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver (None) -> FAIL
 - approver count at least three (0 approvers) -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three (492 commits) -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver (None) -> FAIL
 - approver count at least three (0 approvers) -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> FAIL
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
@mikeller @McGiverGim is this covered?

**Comment by McGiverGim** [comment]
I'm will look into it, usually Crowdin will open it again when new changes

**Pull request closed without merging**